.storybook/webpack.config.js

```diff
@@ -1,9 +1,15 @@
+const path = require('path');
+
 module.exports = function({ config }) {
     config.module.rules.push({
       test: /.story.js$/,
       loaders: [require.resolve('@storybook/addon-storysource/loader')],
       enforce: 'pre',
     });
-  
+
+    config.resolve.alias = {
```

Comment:
@c-harper says this construct might not be something we do long term, but good enough for now.